.pep8

```diff
@@ -0,0 +1,4 @@
+[pycodestyle]
+ignore=E501, W503
```

Comment:
These seem to be related to line length and positioning of line breaks...any reason why we couldn't fix the occurrences of these (or were there too many?)

Reply:
W503(line break before) conflicts with W504(line break after) and W503 seems  like it will eventually be depreciated, so I choose to fix W504 and ignore W503.

Reply:
As for line length, I think that's due to pyline ignore comments, which I don't think can be moved to different line.